src/pyne/update_pyne.sh

```diff
@@ -2,8 +2,9 @@
 
 set -e
 
+branch="develop"
```

Comment:
Is there a benefit to creating a variable here instead of using "develop" directly in the clone command below?

Reply:
I liked it because it allows to play with it easily, and try out other version when necessary.
I can flip that back, don't have a strong filling here.

Reply:
I guess I'd prefer to keep it in the same line unless it's used in multiple places or becomes an external parameter for the script. Less to parse mentally.